### PR TITLE
complete chinese translation

### PR DESCRIPTION
**Pull request by wgzhao**
I complete the unfinished chinese translate

### CODE REVIEW

**Comment by xrmx** [comment]
Looks like this has been superseded by #534 and the repo for this pull request disappeared.

File: caravel/config.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
 # ---------------------------------------------------------
 
 # Your App secret key
-SECRET_KEY = '\2\1thisismyscretkey\1\2\e\y\y\h'  # noqa
+SECRET_KEY = 'FWGVwZRAfFNyjt0hNViNRJ6heBkHqWIwboI0fYgtsGorkiRQhaB3fXuoBKcz4ebA'  # noqa
```
Comment (mistercrunch) [inline caravel/config.py]: oops, you may want to rotate that

File: .idea/inspectionProfiles/Project_Default.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<component name="InspectionProjectProfileManager">
```
Comment (mistercrunch) [inline .idea/inspectionProfiles/Project_Default.xml]: what's this?

**Comment by mistercrunch** [comment]
Let's restart this PR clean, closing

**Pull request closed without merging**